### PR TITLE
Project Updates

### PR DESCRIPTION
**Pull request by paltman**
#### What's this PR do?

* Adds new pinax-stripe badge to README
* Cleans up status badges in README
* Switches over to CircleCI from TravisCI

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+          DJANGO=https://github.com/django/django/tarball/master
+
+workflows:
+  version: 2
+  lint-and-test:
+    jobs:
+      - lint
+      - py27dj18:
+          requires:
+            - lint
```
Comment (blueyed) [inline .circleci/config.yml]: Should "lint" be a requirement really?  (assuming that it will not run the other builds then)

Comment (paltman) [reply .circleci/config.yml]: what that tells circle CI is to run the linting first before starting the testing.

screen shot 2017-09-29 at 10 59 23 am

Comment (blueyed) [reply .circleci/config.yml]: And if linting fails?

Comment (paltman) [reply .circleci/config.yml]: then it doesn't bother running the tests.  you fix your lints and push again.

Comment (blueyed) [reply .circleci/config.yml]: well, that seems to be a bad feedback loop to me.
But:yeah for keeping the project on track!

https://user-images.githubusercontent.com/1192/30399847-a9cd658e-989a-11e7-85e7-264e87b603e7.gif

Comment (paltman) [reply .circleci/config.yml]: Why not fail fast?

**Pull request merged**